src/components/Account.js

```diff
@@ -187,88 +187,84 @@ function Account(props) {
   ));
 
   return (
-    <>
-      <Grid container className={classes.box}>
-        <Grid item xs={3}>
-          <Paper elevation={3} className={classes.menu}>
-            <Menu variant="plain" />
-          </Paper>
-        </Grid>
-        <Grid item xs={9}>
-          <Grid container className={classes.rightBox}>
-            <Grid item xs={12}>
-              <Grid container className={classes.titleBox}>
-                <Grid item>
-                  <AccountIcon className={classes.accountIcon} />
-                </Grid>
-                <Grid item>
-                  <Typography variant="h2">Account</Typography>
-                </Grid>
+    <div style={{ display: 'flex', flexDirection: 'row', flexWrap: 'nowrap' }}>
+      <Paper elevation={3} style={{ width: '300px' }}>
```

Comment:
Since we want a fixed and consistent menu width throughout the tool, it's best to set the width of the outer element of the `<Menu>` component in Menu.js.
There's already a `MENU_WIDTH` value there, but it's only being applied to drawer version of the menu.
If we move the width to the core menu component, it will apply everywhere.